openshift-client/src/main/java/io/fabric8/openshift/client/OpenShiftConfig.java

```diff
@@ -155,4 +205,48 @@ public long getBuildTimeout() {
   public void setBuildTimeout(long buildTimeout) {
     this.buildTimeout = buildTimeout;
   }
+
+  public boolean isNoServer() {
+    return noServer;
+  }
+
+  public void setNoServer(boolean noServer) {
+    this.noServer = noServer;
+  }
+
+  /**
+   * Updates the list items if they have missing or default apiVersion values and the resource is currently
+   * using API Groups with custom version strings
+   */
+  public void updateApiVersion(KubernetesResourceList list) {
```

Comment:
there are 2 methods; one for lists (KubernetesResourceList) and one for resources (HasMetadata) - that covers Template and TemplateList right?